<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EllipsisConnectDialogBase</class>
 <widget class="QDialog" name="EllipsisConnectDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ellipsis Drive Connect</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QRadioButton" name="radioRaster">
      <property name="text">
       <string>Raster data</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_communitysearch"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_wmts">
      <property name="text">
       <string>Get wmts</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_wfs">
      <property name="text">
       <string>Get wfs</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QListWidget" name="listWidget_community"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="radioVector">
      <property name="text">
       <string>Vector data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_wms">
      <property name="text">
       <string>Get wms</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_communitysearch</tabstop>
  <tabstop>radioRaster</tabstop>
  <tabstop>radioVector</tabstop>
  <tabstop>listWidget_community</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
